motorApp/Db/linmot_extra.db

```diff
@@ -0,0 +1,3 @@
+record(motor, "$(P)$(M)") {
+  field(OFF, "$(OFF)")
```

Comment:
Can we call this offset off might be interpreted as turn motor off. Add description field too please.